lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -108,7 +104,7 @@ def instructor_dashboard_2(request, course_id):
         'studio_url': get_studio_url(course, 'course'),
         'sections': sections,
         'disable_buttons': disable_buttons,
-        'analytics_dashboard_message': analytics_dashboard_message
+        'analytics_dashboard_url': analytics_dashboard_url,
```

Comment:
Good point, I'll rename this variable ```insight_dashboard_url```

@jinpa/@stvstnfrd: is it ok if I rename the ```Analytics``` menu item ```Demographics```?